autoload/lsp/diag.vim

```diff
@@ -141,6 +154,24 @@ def DiagsRefresh(bnr: number)
     return
   endif
 
+  # Initialize default/fallback properties for diagnostic virtual text:
+  var diag_align: string = 'above'
+  var diag_wrap: string = 'truncate'
+  var diag_symbol: string = '┌─'
+  var diag_padding: number = 0
```

Comment:
Seems like this variable is a bit useless as you can simply set `padding = 3` for the 'after' condition in the loop?

Reply:
My intention here was to define all *magic* numbers and settings at the same place. But no problem for me to eliminate this variable `diag_padding` if this improves the code.

Reply:
I think it does, as it's the only case `diag_padding` is used is for `after` the two other cases computes the `padding` based on the visual size of the start character. Maybe even move that padding calculation to the else branch:

```
if diag_align == 'after'
  padding = 3
  ...
else
  padding = diag.range.start.character
  if padding > 0
    ...
  endif
endif